src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
 public class WorkWithFile {
 
+    private int supplySum = 0;
+    private int buySum = 0;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        readFromFile(fromFileName);
+        try {
+            Files.createFile(Path.of(toFileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Some troubles with creating a file " + toFileName + e);
+        }
+        writeToFile(toFileName);
+    }
+
+    private void readFromFile(String fromFileName) {
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            int amountIndex = 1;
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                if (line.contains("supply")) {
+                    supplySum += Integer.parseInt(line.split(",")[amountIndex]);
```

Comment:
same for `,` - move it to a constant
also, let's save `line.split(",")` into a separate variable, and then reuse it